docs/user-guide/common-use-cases/assets.md

```diff
@@ -101,9 +101,203 @@ $ curl -X GET http://localhost:8090/v2/wallets/1f82e83772b7579fc0854bd13db6a9cce
+
+Please note that you would mint and burn other assets pretty much the same way. In our example we will be additionally adding on-chain metadata to our minting transaction as we want our NFT to be CIP-25 compliant. However this step is not required for minting. For instance, you could mint some assets and add off-chain metadata for them ([CIP26](https://cips.cardano.org/cips/cip26)) in the [Cardano Token Registry](https://developers.cardano.org/docs/native-tokens/token-registry/cardano-token-registry).
+
+### Minting an NFT
+
+Let's see how can we mint an NFT with [CIP-25](https://cips.cardano.org/cips/cip25/) metadata using `cardano-wallet`.
+
+#### Policy key
+
+Before we attempt any minting/burning transaction we need to make sure our wallet is set up with a policy key. We can check it using [`GET /wallets/{walletId}/policy-key`](https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/getPolicyKey):
+```
+$ curl -X GET http://localhost:8090/v2/wallets/73d38c71e4b8b5d71769622ab4f5bfdedbb7c39d/policy-key
+"policy_vk12d0gdel9u6px8wf3uv4z6m4h447n9qsad24gztaku8dzzdqfajzqfm3rr0"
+```
+Looks good.
+In case we get `missing_policy_public_key` error:
+```
+$ curl -X GET http://localhost:8090/v2/wallets/73d38c71e4b8b5d71769622ab4f5bfdedbb7c39d/policy-key
+{
+  "code": "missing_policy_public_key",
+  "message": "It seems the wallet lacks a policy public key. Therefore it's not possible to create a minting/burning transaction or get a policy id. Please first POST to endpoint /wallets/{walletId}/policy-key to set a policy key."
+}
+
+```
+We just need to make a `POST` request to [`POST /wallets/{walletId}/policy-key`](https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/postPolicyKey) as suggested in the error message.
+
+```
+$ curl -X POST http://localhost:8091/v2/wallets/73d38c71e4b8b5d71769622ab4f5bfdedbb7c39d/policy-key \  
+-d '{"passphrase":"Secure Passphrase"}' \  
+-H "Content-Type: application/json"
+
+"policy_vk12d0gdel9u6px8wf3uv4z6m4h447n9qsad24gztaku8dzzdqfajzqfm3rr0"
+```
+
+Once we sort it out we are all set and we can proceed into minting an NFT from our wallet!
+
+
+#### CIP-25 metadata
+
+We will be attaching CIP-25 metadata to our minting transaction, so there are few things that need to be sorted out. Most basic metadata can look as follows:
+```
+{
+      "721": {
+        "<POLICY_ID>": {
+          "<ASSET_NAME>": {
+            "name": "My amazing NFT",
+            "image": "ipfs://<IPFS_ID>"
+          }
+        }
+      }
+}
+```
+
+As we can see we need to figure out `<POLICY_ID>`, `<ASSET_NAME>` and `<IPFS_ID>`.
+
+##### Policy ID
+
+Policy id is basically a hash of the native script that guards minting operation. In case of Shelley wallets we can only sign with one key, a wallet spending key, but because of the fact that we can embed it into a native script we can practically have unlimited amount of policy ids from one wallet.
```

Comment:
```suggestion
Policy id is basically a hash of the native script that guards minting operation. In case of Shelley wallets we can only sign with one key, a wallet policy key, but because of the fact that we can embed it into a native script we can practically have unlimited amount of policy ids from one wallet.
```